src/polyglot-notebooks-vscode-common/tests/metadataUtilities.test.ts

```diff
@@ -1,1092 +1,1034 @@
 
-        it('cell metadata can be extracted from an interactive document element with old metadata', () => {
-            const interactiveDocumentElement: commandsAndEvents.InteractiveDocumentElement = {
-                contents: '',
+    it("cell metadata can be extracted from an interactive document element with old metadata", () => {
+        const interactiveDocumentElement: commandsAndEvents.InteractiveDocumentElement =
+            {
+                contents: "",
                 outputs: [],
                 executionOrder: 0,
                 metadata: {
                     dotnet_interactive: {
-                        language: 'fsharp'
-                    }
-                }
+                        language: "fsharp",
+                    },
+                },
             };
-            const notebookCellMetadata = metadataUtilities.getNotebookCellMetadataFromInteractiveDocumentElement(interactiveDocumentElement);
-            expect(notebookCellMetadata).to.deep.equal({
-                kernelName: 'fsharp'
-            });
+        const notebookCellMetadata =
+            metadataUtilities.getNotebookCellMetadataFromInteractiveDocumentElement(
+                interactiveDocumentElement
+            );
+        expect(notebookCellMetadata).to.deep.equal({
+            kernelName: "fsharp",
         });
+    });
 
-        it('cell metadata can be extracted from an interactive document element', () => {
-            const interactiveDocumentElement: commandsAndEvents.InteractiveDocumentElement = {
-                contents: '',
+    it("cell metadata can be extracted from an interactive document element", () => {
+        const interactiveDocumentElement: commandsAndEvents.InteractiveDocumentElement =
+            {
+                contents: "",
                 outputs: [],
                 executionOrder: 0,
                 metadata: {
                     polyglot_notebook: {
-                        kernelName: 'fsharp'
-                    }
-                }
+                        kernelName: "fsharp",
+                    },
+                },
             };
-            const notebookCellMetadata = metadataUtilities.getNotebookCellMetadataFromInteractiveDocumentElement(interactiveDocumentElement);
-            expect(notebookCellMetadata).to.deep.equal({
-                kernelName: 'fsharp'
-            });
+        const notebookCellMetadata =
+            metadataUtilities.getNotebookCellMetadataFromInteractiveDocumentElement(
+                interactiveDocumentElement
+            );
+        expect(notebookCellMetadata).to.deep.equal({
+            kernelName: "fsharp",
         });
+    });
 
-        it('cell metadata can be extracted from a notebook cell with old metadata', () => {
-            const notebookCell: vscodeLike.NotebookCell = {
-                kind: vscodeLike.NotebookCellKind.Code,
-                metadata: metadataUtilities.useLegacyMetadata() ? {
-                    custom: {
```

Comment:
I think it would be worthwhile to still allow reading the `custom` property and ensure that the new format is written back out.  This scenario would cover a developer that has an old notebook that only now needs to be updated.

Reply:
This code is not dealing with the file, this was a change in object model in vscode api (@DonJayamanne correct right?) so we needed a way to be coexisting with stable and insiders on different object models. Now we can drop this code.

Reply:
The `custom` property no longer exists in VS Code.
The change to move away from `custom` was made late last year, hence I do not se the need to add a backward compatibility anymore.

> so we needed a way to be coexisting with stable and insiders on different object models. Now we can drop this code.

Yes